src/Dialogs/Settings/dlgBasicSettings.cpp

```diff
@@ -271,9 +282,16 @@ FlightSetupPanel::Prepare(ContainerWindow &parent,
   const ComputerSettings &settings = CommonInterface::GetComputerSettings();
   const Plane &plane = CommonInterface::GetComputerSettings().plane;
 
+  AddFloat(_("Crew"),
+           _("All masses loaded to the glider beyond the empty weight including pilot and copilot, but not water ballast."),
+           _T("%.0f %s"), _T("%.0f"),
+           0, 300, 5, false, UnitGroup::MASS,
+           polar_settings.glide_polar_task.GetCrewMass(),
+           this);
+  
   const double db = 5;
   AddFloat(_("Ballast"),
-           _("Ballast of the glider.  Increase this value if the pilot/cockpit load is greater than the reference pilot weight of the glide polar (typically 75kg).  Press ENTER on this field to toggle count-down of the ballast volume according to the dump rate specified in the configuration settings."),
+           _("Ballast of the glider. Press ENTER on this field to toggle count-down of the ballast volume according to the dump rate specified in the configuration settings."),
```

Comment:
What does "ENTER" mean? The "ENTER" key? Most devices don't have an "ENTER" key; wording should be portable, not just the code.

Reply:
Not introduced through my change. I am responsible for the remove of the first part of the help message, which is now valid for the crew load.

Reply:
Oh, right! Sorry, this is an old bad message.

Reply:
Replaced the ENTER in the help text.